### PR TITLE
Add explicit references article

### PR DESCRIPTION
**Pull request by zacjones93**
Inspired by [this article](https://www.a11ywithlindsey.com/blog/accessibility-conferences)

I would love suggestions and edits welcome!

### CODE REVIEW

File: content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+date: 2019-11-27
+title: 'Accessible Speech'
+description: 'Learn how to use explicit references in your speech for clearer teaching'
+categories: ['instructor', 'style']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_style.png'
```
Comment (zacjones93) [inline content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx]: Just used a share image from another file, not sure if there's a list that I could pull from

File: content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+redirect_from:
+  - /accessible-speech/
```
Comment (zacjones93) [inline content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx]: Also copied this from another file, does the root level get created for every article?

File: content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - /accessible-speech
+---
+
+An egghead instructor, [Lindsey Kopacz](https://twitter.com/LittleKope), published a blog post on [Accessibility at Conferences as a speaker](https://www.a11ywithlindsey.com/blog/accessibility-conferences) which has quite a few great points in general as well as some that will help with screen casts. Check it out.
```
Comment (zacjones93) [inline content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx]: Is this the best way to give Lindsey credit?

File: content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx (hunk 2 above)
Comment (joelhooks) [reply content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx]: This is only needed if it was a previously published article that you moved.

Comment (joelhooks) [reply content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx]: This is only needed if it was a previously published article that you moved.

**Pull request merged**